integrations/dalle/integration.definition.ts

```diff
@@ -0,0 +1,37 @@
+import { IntegrationDefinition, messages } from '@botpress/sdk'
+import { z } from 'zod'
+import { name } from './package.json'
+
+export default new IntegrationDefinition({
+  name,
+  version: '0.2.0',
+  icon: 'icon.svg',
+  title: 'Dalle Image Generation',
+  description: 'Generate images using Dalle',
+  readme: 'hub.md',
+  configuration: {
+    schema: z.object({
+      apiKey: z.string().describe('Open API Key'),
+    }),
+  },
+  channels: {
+    channel: {
```

Comment:
**chore:** Dalle has no messaging channel. Use the following code:

```ts
export default new IntegrationDefinition({
// ...
  channels: {}
// ...
})
```

this will allow you to remove the unimplemented message handlers in `src/index.ts`.